extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -46,6 +46,16 @@ export const ReactionType = {
   POLL: 1,
 };
 
+/**
+ * @const @enum {string}
+ */
+export const ReactionFlags = {
+  THEME: 'theme',
+  CHIP_CORNER: 'chip-corner',
+  CHIP_STYLE: 'chip-style',
+  STYLE: 'style',
```

Comment:
We need to bring the style from the element to the shadow root to keep the variables, would be an improvement to have attributes "background-color" and "accent-color" to specifically override those styles and allow nothing else. However, it's not in the scope of this PR